packages/front-end/components/Settings/EditDataSourceSettingsForm.tsx

```diff
@@ -269,6 +280,20 @@ const EditDataSourceSettingsForm: FC<{
                               )
                             }
                           />
+                          <div>
+                            <label style={{ padding: 3 }}>
```

Comment:
Don't add inline padding to the label.  Instead, you can add a class to the outer div `form-group`.  This will apply the same padding/margin styles that are used everywhere else on the page.

Reply:
Should there be some space between the toggle and the text/tooltip?